service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsSubmittedControllerTest.java

```diff
@@ -59,7 +58,8 @@
     private static final String LOCAL_AUTHORITY_CODE = "example";
     private static final Long CASE_ID = 12345L;
     private static DocumentReference applicationDocument;
```

Comment:
probably not relevant for this test, can be removed or if still want to have it for data completness, can be hidden in buildCommonNotificationParameters